crates/wasi-common/src/snapshots/wasi_snapshot_preview1.rs

```diff
@@ -662,12 +651,8 @@ impl<'a> WasiSnapshotPreview1 for WasiCtx {
             new_path,
             true,
         )?;
-        let old_path = unsafe {
-            let mut bc = GuestBorrows::new();
-            let raw = old_path.as_raw(&mut bc)?;
-            &*raw
-        };
-        trace!("     | old_path='{}'", old_path);
+        let old_path = old_path.as_str()?;
+        trace!("     | old_path='{}'", &*old_path);
```

Comment:
I know it's not saving us much, but just FWIW, I think this one-liner should also work:

```rust
trace!("     | old_path='{}'", &*old_path.as_str()?);
```

Reply:
We need it in the next line as a &str, too